docs/source/index.md

```diff
@@ -1,9 +1,29 @@
-# Introduction
+🚧 Under Active Development 🚧
 
+# 🦜💪 LangChain Benchmarks
+
+A package to help benchmark various LLM related tasks.
+
+The benchmarks are organized by end-to-end use cases, and
+utilize [LangSmith](https://smith.langchain.com/) heavily.
+
+We have several goals in open sourcing this:
+
+- Showing how we collect our benchmark datasets for each task
+- Showing what the benchmark datasets we use for each task is
+- Showing how we evaluate each task
+- Encouraging others to benchmark their solutions on these tasks (we are always looking for better ways of doing things!)
+
+We currently include the following tasks:
+- [CSV Question Answering](https://github.com/langchain-ai/langchain-benchmarks/tree/main/csv-qa)
+- [Extraction](https://github.com/langchain-ai/langchain-benchmarks/tree/main/extraction)
+- [Q&A over the LangChain docs](https://github.com/langchain-ai/langchain-benchmarks/tree/main/langchain-docs-benchmarking)
+- [Meta-evaluation of 'correctness' evaluators](https://github.com/langchain-ai/langchain-benchmarks/tree/main/meta-evals)
 
 ```{toctree}
 :maxdepth: 2
 :caption: Contents
+:glob:
```

Comment:
We'll remove the glob later, for now it's just easy to make sure all content appears